app/lib/pdf_filler/md502_su_pdf.rb

```diff
@@ -0,0 +1,36 @@
+module PdfFiller
+  class Md502SuPdf
+    include PdfHelper
+
+    def source_pdf_name
+      "md502SU-TY2023"
+    end
+
+    def initialize(submission)
+      @submission = submission
+      builder = StateFile::StateInformationService.submission_builder_class(:md)
+      @xml_document = builder.new(submission).document
```

Comment:
[pebble] similar to previous comment -- doesn't look like this instance variable is referenced anywhere in this class. could it be removed?